src/main/kotlin/app/UserCtrl.kt

```diff
@@ -62,8 +63,9 @@ object UserCtrl {
     }
 
     private fun getYearAndQuarter(it: RepositoryCommit): String {
-        val date = it.commit.committer.date
-        return "${(1900 + date.year)}-Q${date.month / 3 + 1}"
+        val cal = Calendar.getInstance()
```

Comment:
val date = it.commit.committer.date.toInstant().atZone(ZoneId.systemDefault()).toLocalDate()
return "${date.year}-Q${(date.get(IsoFields.QUARTER_OF_YEAR))}"